glean-core/src/database/mod.rs

```diff
@@ -2,8 +2,11 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+use chrono::{Local, NaiveDate};
+use lazy_static::lazy_static;
```

Comment:
I personally prefer to split out crate uses from std uses, so I would suggest moving that down to line 14 (old line 11)